src/ServiceControlInstaller.Engine/Instances/ServiceControlInstance.cs

```diff
@@ -111,11 +113,7 @@ public string StorageUrl
                         host = HostName;
                         break;
                 }
-                if (string.IsNullOrWhiteSpace(VirtualDirectory))
-                {
-                    return $"http://{host}:{Port}/storage/";
-                }
-                return $"http://{host}:{Port}/{VirtualDirectory}{(VirtualDirectory.EndsWith("/") ? String.Empty : "/")}storage/";
+                return $"http://{host}:{MaintenancePort}/";
```

Comment:
Should we take the virtual directory into account?

Reply:
> This setting is provided for backward compatibility and should be considered obsolete.

https://docs.particular.net/servicecontrol/creating-config-file#host-settings-servicecontrolvirtualdirectory

I'd say no